akka-cluster-typed/src/main/scala/akka/cluster/typed/ClusterSingleton.scala

```diff
@@ -124,6 +124,56 @@ private[akka] object ClusterSingletonImpl {
   def managerNameFor(singletonName: String) = s"singletonManager$singletonName"
 }
 
+object Singleton {
```

Comment:
Didn't see a need to break out a separate java/scaladsl WDYT?

Reply:
I don't think it's a good name, because Scala will have `scala.Singleton`.

Reply:
good catch

`SingletonActor` ?

Reply:
Yeah pity we can't call it `ClusterSingleton` :)

Reply:
Can't come up with a better name :(